skipper.go

```diff
@@ -1309,9 +1309,8 @@ func listenAndServeQuit(
 	}
 
 	if o.EnableConnMetricsServer {
-		m := metrics.Default
```

Comment:
minor: why were `metrics.Default` and `mtr` were mixed in this function at all?

Reply:
:shrug: